code/modules/assembly/flash.dm

```diff
@@ -114,7 +114,7 @@
 		var/mob/living/silicon/robot/R = M
 		add_logs(user, R, "flashed", src)
 		update_icon(1)
-		M.Weaken(6)
+		M.Weaken(2)

```

Comment:
That is pretty extreme. That would mean it was even **worse** than the current flash. Note, did you check that Weaken() did the same thing to silicons back then as it does now?